presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -393,7 +394,10 @@ protected void setup(Binder binder)
         binder.bind(FinalizerService.class).in(Scopes.SINGLETON);
 
         // Spiller
-        binder.bind(SpillerFactory.class).to(BinarySpillerFactory.class).in(Scopes.SINGLETON);
+        install(installModuleIf(
```

Comment:
Yep, `NetworkTopology` is how we used to make things extensible, but now we're going with the approach using for security and resource groups, as it allows extensibility via normal plugins, rather than re-building the server